packages/react-data-grid-react-window/stories/DataGrid/VirtualizedDataGrid.stories.tsx

```diff
@@ -241,3 +243,82 @@ VirtualizedDataGrid.parameters = {
     },
   },
 };
+
+const getStylesForReactWindowOverrides = makeStyles({
+  header: {
+    paddingRight: 'var(--scrollbar-width, 4px)',
+    height: 'var(--header-height)',
+  },
+  list: {
+    scrollbarGutter: 'stable',
+  },
+  grid: {
+    height: '100%',
+  },
+});
+
+export const ReactWindowOverrides = () => {
```

Comment:
There is no automation to enforce this yet, but move this example to a separate `.stories.tsx` file, otherwise the code sample on the published storybook can be quite funny

Reply:
Done :)